src/overlays/actors/ovl_Bg_Mizu_Water/z_bg_mizu_water.h

```diff
@@ -6,9 +6,15 @@
 
 struct BgMizuWater;
 
+typedef void (*BgMizuWaterActionFunc)(struct BgMizuWater*, GlobalContext*);
+
 typedef struct BgMizuWater {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x14];
+    /* 0x014C */ BgMizuWaterActionFunc actionFunc;
+    /* 0x0150 */ s32 type;
+    /* 0x0154 */ f32 targetY;
+    /* 0x0158 */ f32 baseY;
+    /* 0x015C */ s32 switchTriggerID; // only used for types 2-4
```

Comment:
`triggerSwitchFlag` ?

Reply:
It's the switch ID for the types 2-4 for changing the water level, since type 0 checks all three switches `0x1C`, `0x1D`, `0x1E`. Maybe just `switchID` ?

Reply:
I guess leaving out `trigger` is fine. All actors ever do with switch flags is set/clear/react to them anyway so I'd say it's good 👍